### ISSUE TITLE
Load JDBC drivers in scripts with arguments

### ISSUE THREAD

**Issue by DavidPerezIngeniero** [lihaoyi/Ammonite#462]
I'm creating a simple script that uses JDBC with a Postgres driver.

If I do it like this:

``` scala

import $ivy.`org.postgresql:postgresql:9.4.1209.jre7`
import java.sql.DriverManager

@main
def main() {
     ....
    val con = DriverManager.getConnection("jdbc:postgresql://localhost/mydatabase", "user", "password")
    ....
}
```

then it fails like this:

```
java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost/mydatabase
```

even though if I add before this line:

```
classOf[org.postgresql.Driver].getName
```

After some experimentation, I've found that the solution is this:

``` scala
def main() {
     ....
    val con = DriverManager.getConnection("jdbc:postgresql://localhost/mydatabase", "user", "password")
    ....
}
main()
```

Loading the driver isn't necessary with `Class.forName()` or `classOf[org.postgresql.Driver].getName`.

**Conclusion**: 

If I want to use arguments in my script, then JDBC drivers aren't used, even if they are loaded in the JVM.

I consider this behaviour a bug in Ammonite.

Any workaround?

**Comment by DavidPerezIngeniero** [comment]
I think it has to do with some Class loader magic when using arguments.

**Comment by lihaoyi** [comment]
Odd, not sure what's happening, do you think you could help investigate and fix it?

**Comment by DavidPerezIngeniero** [comment]
Tried to place the import inside the main method, but it doesn't like it:

    object org.postgresql:postgresql:9.4.1209.jre7 is not a member of package $ivy
            import $ivy.`org.postgresql:postgresql:9.4.1209.jre7`

**Comment by DavidPerezIngeniero** [comment]
Found a workaround:

Call inside main method this:

    Class.forName("org.postgresql.Driver")

It isn't need it, when using no main method.

**Comment by lihaoyi** [comment]
Should be the same problem as https://github.com/lihaoyi/Ammonite/issues/591, which was fixed in 0.8.5
